ansible/templates/minigraph_cpg.j2

```diff
@@ -80,20 +80,40 @@
 {% endfor %}
 {% endif %}
 {% if switch_type is defined and switch_type == 'voq' %}
-{% for all_idx in range(all_inbands|length) %}
-{% if voq_inband_ip != all_inbands[all_idx] %}
+{% set voq_peers = dict() %}
+{% for asic_id in range(num_asics) %}
+{% if num_asics == 1 %}
+{% set start_rtr = inventory_hostname %}
+{% else %}
+{% set start_rtr = "ASIC" + asic_id|string %}
```

Comment:
start_rtr also requires the linecard/hostname prefix before the "ASIC"

Reply:
No, because is BGPRouterDeclaration for each asic (lines 221-226 in the file, we are defining the hostname as 'ASIC<#>'.

Reply:
ASIC<#> is not unique for the whole chassis, but within a linecard. In this section, we are creating iBGP peers for a linecard, so for a multi-asic linecard, the starting router includes only the ASIC<#>. The end_rtr includes the other linecards hostname and ASIC<#> which is unique for the whole chassis.  

This is similar to iBGP defined at [L44](https://github.com/Azure/sonic-buildimage/blob/66089647b9da5764e4abf00e50ecfd745cc65f62/src/sonic-config-engine/tests/multi_npu_data/sample-minigraph.xml#L44) in the sample multi-asic xml file.